types/react-table/index.d.ts

```diff
@@ -795,7 +799,11 @@ export namespace useSortBy {
     const pluginName = 'useSortBy';
 }
 
-export interface TableSortByToggleProps {}
+export interface TableSortByToggleProps {
+    title?: string | undefined;
+    style: CSSProperties;
```

Comment:
Can you please confirm that this is not supposed to be optional (e.g. as on `TableCommonProps`)?

Reply:
Same for `onClick`.

Reply:
see:

https://github.com/tannerlinsley/react-table/blob/master/src/plugin-hooks/useSortBy.js#L33-L54
https://github.com/tannerlinsley/react-table/blob/master/src/plugin-hooks/useGroupBy.js#L37-L51

That said, since none of these values are documented anywhere (so might change) I think that I'm going to make them all be optional since I think that that reflects the contract (or lack thereof) better.

Reply:
Thanks!